bin/github-draft-release.sh

```diff
@@ -129,14 +106,4 @@ elif [[ "$current" != "$latest" ]]; then
 		fi
 		gh release create "$tag" --draft --generate-notes --target "$commit" --title "$tag" "$package"
 	fi
-else # if [[ -n "$draft" ]] && [[ "$current" == "$latest" ]]; then
```

Comment:
I think we should add a "please publish release before creating new draft".

Overwriting the draft is bad, we would lose the commit the NPM package was published with.